sympy/combinatorics/pc_groups.py

```diff
@@ -231,24 +231,12 @@ def collected_word(self, word):
             if low == -1:
                 continue
             s1, e1 = w[0]
-            if len(w) == 2 and w[1][1] > 0:
-                s2, e2 = w[1]
-                s2 = ((s2, 1), )
-                s2 = free_group.dtype(s2)
-                word_ = self.map_relation(free_group.dtype(w))
-                word_ = s2*word_**e1
-                word_ = free_group.dtype(word_)
-                word = word.substituted_word(low, high, word_)
 
-            if not self.relative_order[self.index[s1]]:
-                continue
             re = self.relative_order[self.index[s1]]
```

Comment:
It should be first checked that `len(w) == 1`. If `len(w) == 2`, relative order is not needed.